CHANGELOG.md

```diff
@@ -1,58 +1,63 @@
+- `Rint`, `Rdouble`, `Rbool` and `Rcplx`: `ALTREP` wrapper classes for R vectors, which can be used without copying [[#274]](https://github.com/extendr/extendr/pull/274), [[#284]](https://github.com/extendr/extendr/pull/284), [[#301]](https://github.com/extendr/extendr/pull/301), [[#338]](https://github.com/extendr/extendr/pull/338), [[#350]](https://github.com/extendr/extendr/pull/350)
+- `Integers`, `Doubles`, `Strings`, `Logicals` and `Complexes`: `ALTREP` wrappers for vectors that allows for much larger vectors than native R [[#274]](https://github.com/extendr/extendr/pull/274), [[#284]](https://github.com/extendr/extendr/pull/284), [[#301]](https://github.com/extendr/extendr/pull/301), [[#338]](https://github.com/extendr/extendr/pull/338), [[#350]](https://github.com/extendr/extendr/pull/350)
+- `ExternalPtr`, a wrapper class for creating R objects containing any Rust object. [[#260]](https://github.com/extendr/extendr/pull/260)
+- Support for R graphics and graphics devices. These are locked behind the `graphics` feature flag, which is disabled by default. [[#279]](https://github.com/extendr/extendr/pull/279), [[#360]](https://github.com/extendr/extendr/pull/360), [[#373]](https://github.com/extendr/extendr/pull/373), [[#379]](https://github.com/extendr/extendr/pull/379), [[#380]](https://github.com/extendr/extendr/pull/380), [[#389]](https://github.com/extendr/extendr/pull/389)
+- Implemented `Deref` for vector types (Rint/Rfloat/Rbool/Rstr/Robj) to appropriately typed Rust slices [[#327]](https://github.com/extendr/extendr/pull/327)
+- Added a `default` option for `extendr`-annotated functions, allowing them to have default values, e.g. `fn fred(#[default="NULL"] x: Option><i32>) { }` [[#334]](https://github.com/extendr/extendr/pull/334)
+- The `r_name` option for `extendr`-annotated functions, allowing the generated R function to have a different name. e.g.
+    ```rust
+    #[extendr(
+        use_try_from = true,
+        r_name = "test.rename.rlike",
+        mod_name = "test_rename_mymod"
+    )]
+    fn test_rename() { }
+    ```
+    [[#335]](https://github.com/extendr/extendr/pull/335)
+- Implemented `serde::Serialize` for R types [[#305]](https://github.com/extendr/extendr/pull/305), [[#355]](https://github.com/extendr/extendr/pull/355)
+- `Rany` type and the `as_any` conversion method. [[#320]](https://github.com/extendr/extendr/pull/320)
+- Implemented `std::fmt::Debug` for wrapper types [[#345]](https://github.com/extendr/extendr/pull/345)
+- Implemented the `[[` operator to function on Rust classes, where it functions the same the `$` operator. [[#359]](https://github.com/extendr/extendr/pull/359)
+- `Load` and `Save`, traits that implement the ability to load and save R data to the RDS format. These traits are implemented for all `Robj` [[#363]](https://github.com/extendr/extendr/pull/363)
```

Comment:
Not sure I agree with the phrase `"traits that implement"`. Traits by themselves do not implement anything (at least in the majority of cases), they are more like an API contract. So I'd propose 
```suggestion
- `Load` and `Save`, traits that, once implemented, provide the ability to load and save R data in the RDS format. These traits are implemented for all `Robj` [[#363]](https://github.com/extendr/extendr/pull/363)
```